highfidelity/hifi
# Add textures BC compression

- Add NVTT library
- Create new image library
- Add gamma-corrected mip generation
- Add mip compression (Disabled in interface)
- Add compressed texture support to the gpu library

## Test Plan:
- With a clean cache (Delete `AppData/Local/High Fidelity - PR10215`)
- Go to Welcome
- Check that the textures look the same as on master.
- Some specific textures to check:
    - The transparent blue glow of the portals.
    - The leaf on the trees
    - The rocky floor in Playa
- Standard smoke test
- Measure load times from clean cache for Welcome, Zaru, Playa with master and this PR.
